### ISSUE TITLE
Long formula can break layout on narrow screens

### ISSUE THREAD

**Issue by skalee** [geolexica/geolexica-server#141]
See pictures below:

![narrow-formula-broken-1](https://user-images.githubusercontent.com/154287/97716646-2b760880-1ac4-11eb-9595-5b0ba1ac12aa.png)
![narrow-formula-broken-2](https://user-images.githubusercontent.com/154287/97716635-26b15480-1ac4-11eb-8203-a87cc65374e8.png)

**Comment by ronaldtse** [comment]
Yes this is a known problem with MathJax. We need to find some solution. Maybe breaking the MathML... either in markup or in rendering?

Ping @opoudjis if you have any solution.

**Comment by opoudjis** [comment]
Refer to the discussion ion https://github.com/metanorma/metanorma-standoc/issues/111. I don't have a solution that satisfies me (because it isn't supported in AsciiMath), but the investigation does discuss what MathML tends to do.
